content/blog/announcing_kubeflow_0.3.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Kubeflow 0.3 Simplifies Setup & Improves ML Development"
+description = "Announcing Kubeflow 0.3"
+weight = 20
+publishDate = 2018-10-26T11:52:19-07:00
+draft = false
++++
+
+## Since Last We Met
+
+It has been 3 months since landing [0.2](https://www.kubeflow.org/blog/announcing_kubeflow_0.2/) and we have continued to be amazed by the support from the community. The community continues to grow, with 100+ members from 20 different organizations and more than 4,500 Github stars!
+
+In September, we held our first [contributor summit](http://bit.ly/kf-summit-2018-overview)!
+
+ 
+We were thrilled to have over 60 participants join in person along with 10 remote contributors from 4 continents! We are tentatively planning our next summit for early 2019.
+
+To all our contributors, we want to share our deep appreciation for your support! We are getting closer to our vision of letting data scientists and ML engineers focus on building models and products by giving them a Kubernetes native platform for ML that is  easy-to-use, portable and  scalable.
+
+## Upcoming Events
+We have a number of upcoming events where you can connect with us and learn more about Kubeflow
+
+* [KubeCon, Seattle](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/), 11-13 December, 2018
+* [YOW!, Melbourne](https://melbourne.yowconference.com.au/), 4-7 December, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://melbourne.yowconference.com.au/proposal/?id=6858): Michelle Casbon
+* [YOW!, Brisbane](https://brisbane.yowconference.com.au/), 3-4 December, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://brisbane.yowconference.com.au/proposal/?id=6859): Michelle Casbon
+* [YOW!, Sydney](https://sydney.yowconference.com.au/), 27-30 November, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://sydney.yowconference.com.au/proposal/?id=6860): Michelle Casbon
+* [KubeCon, Shanghai](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/), 14-15 November, 2018
+
+For the latest information please refer to our [events calendar](https://www.kubeflow.org/docs/about/events/).
+
+## Introducing Kubeflow 0.3
+
+We're thrilled to announce the availability of Kubeflow 0.3. This release provides streamlined ways to deploy Kubeflow with increased flexibility in existing components and better multi-framework support. Let's walk through some of the highlights:
+
+<span style="color:#333333;"><span style="background-color:#f5f5f5;">We're thrilled to announce the availability of Kubeflow 0.3. This release provides a streamlined and extensible paradigm for Kubeflow deployment, increased flexibility in existing components and continues to extend the multi-framework support.</span></span>
+
+### Deploy

```

Comment:
What about "Easier Deployment" instead of "Deploy"?

Reply:
Done.